custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js

```diff
@@ -93,7 +93,7 @@ function ServiceDeliveryDashboardController($scope, $http, $location, $routePara
     }
 
     function renderHeaderTooltip(header, tooltipContent) {
-        return '<i class="fa fa-info-circle headerTooplip" style="float: right;" ><div class="headerTooplipText">' + tooltipContent + '</div></i><span>' + header + '</span>';
+        return '<i class="fa fa-info-circle headerTooltip" style="float: right;" ><div class="headerTooltipText">' + tooltipContent + '</div></i><span>' + header + '</span>';
```

Comment:
Not a blocker,
But I think using [this](https://www.w3schools.com/js/js_htmldom_nodes.asp) for creating the html DOM would be much more readable and less prone to such errors.

Reply:
Agreed that concatenating HTML bits is hard to read. Another alternative is to use underscore templates [like this](https://github.com/dimagi/commcare-hq/blob/73f7f914761f93b4751bdc05d0520ccca49ae4eb/corehq/apps/data_interfaces/static/data_interfaces/js/find_by_id.js#L66-L68).